sdk/nodejs/output.ts

```diff
@@ -337,7 +327,7 @@ async function applyHelperAsync<T, U>(value: T, isKnown: boolean, isSecret: bool
     // preserve secretness from our original Output to the new one we're creating.
     return {
         value: transformed,
-        isKnown: considerValueKnown,
+        isKnown: true,
```

Comment:
i hate to ask, but could `isKnown` become an *entirely* computed value based *entirely* on the `val` of the Output?

Reply:
Not without changing everything that creates an output to ensure that all unknown values pass in first-class unknowns.